test/victoria-logs-multilevel_test.go

```diff
@@ -0,0 +1,45 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/stretchr/testify/require"
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+// TestVictoriaLogsMultilevelInstallDefault tests that the %s chart can be installed with default values.
+func TestVictoriaLogsMultilevelInstallDefault(t *testing.T) {
+	t.Parallel()
+	name := "victoria-logs-multilevel"
+	cp := chartInstall(t, name, map[string]string{
+		"storageNodes[0]": "http://example.com:9428",
+	})
+	ctx := context.Background()
+	defer chartCleanup(t, ctx, cp)
+
+	// Verify log-storage StatefulSet was created and is ready using manual polling
+	releaseName := cp.releaseName
+	o := cp.opts
+	namespaceName := o.KubectlOptions.Namespace
+	logSelectDeploymentName := fmt.Sprintf("%s-%s-vlselect", releaseName, name)
+	var logSelectDeployment *appsv1.Deployment
+	err := wait.PollUntilContextTimeout(ctx, pollingInterval, pollingTimeout, true, func(ctx context.Context) (done bool, err error) {
+		logSelectDeployment, err = cp.client.AppsV1().Deployments(namespaceName).Get(ctx, logSelectDeploymentName, metav1.GetOptions{})
+		if err != nil {
+			return false, nil
+		}
+		// Ensure all replicas are ready
+		return logSelectDeployment.Status.ReadyReplicas == *logSelectDeployment.Spec.Replicas && *logSelectDeployment.Spec.Replicas > 0, nil
+	})
+	require.NoError(t, err)
+	require.NotNil(t, logSelectDeployment)
```

Comment:
Why not `k8s.WaitUntilDeploymentAvailable(t, k8sOpts, logSelectDeploymentName, retries, pollingInterval)`?